components/brave_wallet/browser/cardano/cardano_transaction_unittest.cc

```diff
@@ -0,0 +1,232 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/cardano/cardano_transaction.h"
+
+#include <utility>
+
+#include "base/strings/string_number_conversions.h"
+#include "brave/components/brave_wallet/browser/cardano/cardano_rpc_schema.h"
+#include "brave/components/brave_wallet/common/test_utils.h"
+#include "testing/gmock/include/gmock/gmock.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+using testing::_;
```

Comment:
this is prohibited by the styleguide https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
@bridiver what is prohibited?

Reply:
nvm, I misread this.